prov/efa/src/rxr/rxr_ep.c

```diff
@@ -998,10 +998,11 @@ static int rxr_ep_ctrl(struct fid *fid, int command, void *arg)
 		 * shared memory region.
 		 */
 		if (ep->use_shm) {
-			ret = rxr_raw_addr_to_smr_name(ep->core_addr, shm_ep_name);
+			ret = rxr_raw_addr_to_smr_name(ep->core_addr, &shm_ep_name);
 			if (ret < 0)
 				goto out;
 			fi_setname(&ep->shm_ep->fid, shm_ep_name, sizeof(shm_ep_name));
```

Comment:
This is now size of a pointer (before was 256) and is probably not what you want

Reply:
Thanks! The new revision used a different approach. so shm_ep_name is still an array.